.gitattributes

```diff
@@ -0,0 +1,2 @@
+package.json text eol=lf
+package-lock.json text eol=lf
```

Comment:
Das hier sollte die Probleme mit Git und `npm install` lösen, also danach sollte in Git nichts mehr als geändert markiert sein. Falls das immer noch nicht klappt, bitte nochmal Bescheid geben!